server/pb_hooks/pages/index.ejs

```diff
@@ -4,4 +4,173 @@
 </div>
 <div class="w-1/1">
   <%- include('specials.ejs') %>
+</div>
```

Comment:
I didn't make this a partial because it's static and such a big part of the homepage, but could make a partial